.eslintrc.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
 		'indent': 'off',
 		'@typescript-eslint/indent': ['error', 'tab'],
 		'no-multi-spaces': ['error', {ignoreEOLComments: false}],
-		'no-mixed-spaces-and-tabs': 'error'
+		'no-mixed-spaces-and-tabs': 'error',
+		'react/jsx-max-props-per-line': 'off'
```

Comment:
I don't like the rule of X number of props per line, if the prop name or value are smallish then it's more than fine to have them in the same line :)